.github/paths-labeller.yml

```diff
@@ -8,18 +8,6 @@
   - "Feature:ExpressionLanguage":
     - "src/plugins/expressions/**/*.*"
     - "src/plugins/bfetch/**/*.*"
-  - "Team:apm":
-    - "x-pack/plugins/apm/**/*.*"
-    - "x-pack/test/apm_api_integration/**/*.*"
-    - "packages/kbn-apm-synthtrace/**/*.*"
-    - "packages/kbn-apm-synthtrace-client/**/*.*"
-    - "packages/kbn-apm-utils/**/*.*"
   - "Team:Fleet":
     - "x-pack/plugins/fleet/**/*.*"
     - "x-pack/test/fleet_api_integration/**/*.*"
-  - "Team:uptime":
-    - "x-pack/plugins/synthetics/**/*.*"
-    - "x-pack/plugins/ux/**/*.*"
-    - "x-pack/plugins/observability/public/components/shared/exploratory_view/**/*.*"
-  - "Team:obs-ux-management":
```

Comment:
@shahzad31 they're not being removed - it's how git is showing it because they are being moved "up" - you'll see they are "added" in the change too
![image](https://github.com/elastic/kibana/assets/70908889/d5d863f5-e47e-4f5e-9c89-133f5e88925b)